infrastructure/main.tf

```diff
@@ -31,15 +31,6 @@ data "azurerm_user_assigned_identity" "em-shared-identity" {
   resource_group_name = "managed-identities-${var.env}-rg"
 }
 
-resource "azurerm_application_insights" "appinsights" {
```

Comment:
do we want to delete appinsight?

Reply:
Yes. As mentioned with details in JIRA ticket